snapcraft/internal/project_loader/_config.py

```diff
@@ -255,6 +256,20 @@ def _process_remote_parts(self, snapcraft_yaml):
         return snapcraft_yaml
 
 
+def _patchelf_install_required(project_options) -> bool:
+    is_xenial = False
+    with contextlib.suppress(os_release.errors.OsReleaseCodenameError):
+        release_codename = os_release.OsRelease().version_codename()
+        is_xenial = release_codename == 'xenial'
+
+    is_snap = common.is_snap()
+    is_environment = os.getenv('SNAPCRAFT_NO_PATCHELF')
```

Comment:
I realize this isn't new behavior here, but `is_environment` will be true even if `SNAPCRAFT_NO_PATCHELF` is defined to be e.g. 0, or pretty much any other falsy value. Is that intended?

Reply:
Given that this is only intended for testing, I don't mind whatever addition you'd like to do ;-)

Reply:
Nah, no need to hold things up. For future reference, `distutils.util.strtobool` is nifty.